gnosis_vpn-lib/src/session/capability.rs

```diff
@@ -0,0 +1,7 @@
+use serde::Serialize;
+
+#[derive(Clone, Debug, Serialize, Hash)]
+pub enum Capability {
+    Segmentation,
+    Retransmission,
+}
```

Comment:
Isn't there too much repetition, this was also in the config.

Reply:
It sure looks like it, but I would like to keep those types separated.

One of them is the internal library type and the other the external facing configuration type. They might change independently from each other for good reasons.